examples/tutorials/05_train_sklearn_models.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>Why is there no link to Log reg? Be consistent</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>